docs_nnx/guides/surgery.ipynb

```diff
@@ -318,9 +325,9 @@
    "source": [
     "### Naive partial initialization\n",
     "\n",
-    "You can simply initialize the whole model, then swap pre-trained parameters in. But this approach could allocate additional memory midway, if your modification requires re-creating module parameters that you will later discard. See this example below.\n",
+    "To do naive partial initialization, you can just initialize the whole model, then swap the pre-trained parameters in. However, this approach may allocate additional memory midway if your modification requires re-creating module parameters that you will later discard. Below is an example of this.\n",
     "\n",
-    "> Note: You can use `jax.live_arrays()` to check all the arrays live in memory at any given time. This call can be messed up when you run a single notebook cell multiple times (due to garbage-collecting old python variables), but restarting the kernel and running from scratch will always yield same output."
+    "> **Note:** You can use `jax.live_arrays()` to check all the arrays live in memory at any given time. This call can be “messed up” when you run a single Jupyter notebook cell multiple times (because of old garbage-collecting Python variables). However, restarting the Python kernel in the notebook and running the code from scratch will always yield the same output."
```

Comment:
"because of old garbage-collecting Python variables" -> "due to garbage-collection of old Python variables"

Reply:
good catch, thanks! done